### PR TITLE
Completed typescript-template-literals exercise

### PR DESCRIPTION
**Pull request by techmatlock**
Screenshot 2024-05-13 at 6 05 05 PM

### CODE REVIEW

File: typescript-template-literals/main.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+const age = 25;
+
+const bio = `My name is ${firstName} ${lastName} and I am ${age} years old.`;
+
+console.log('Bio:', bio);
+
+const fruit = 'apple';
+const item = 'pen';
+
+const applePen = fruit + item;
```
[the remark this answers is not in the document]
Comment (techmatlock) [reply typescript-template-literals/main.ts]: @brandon-moy Ok, let me submit a new PR.  Thanks.

[the remark this answers is not in the document]
Comment (techmatlock) [reply typescript-template-literals/main.ts]: Actually I see that I can resubmit with this same PR.

**Review by brandon-moy — APPROVED** [review]
Approved!

File: typescript-template-literals/main.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (brandon-moy) [reply typescript-template-literals/main.ts]: If the PR is still open with the branch, when you push commits it adds to the PR 👍

**Pull request merged**